### PR TITLE
:bug: Source Google Ads: Remove metrics from ad group for manager account

### PR DESCRIPTION
**Pull request by darynaishchenko**
## What
Resolved:https://github.com/airbytehq/airbyte/pull/33494#issuecomment-1887744404
The update to add a new field to the ad_group stream is leading to errors for users with the connector set up using a manager account. This is because metrics are not accessible for manager accounts

## How
Removed metrics.cost_micros from Ad Group Stream query for manager account.

## 🚨 User Impact 🚨
Users with manager account will sync Ad Group stream without query error.

### CODE REVIEW

**Comment by tolik0** [comment]
@darynaishchenko, Could you please add a unit test to check if non-manager streams don't have metrics in fields to avoid such issues in the future?

**Comment by darynaishchenko** [comment]
> @darynaishchenko, Could you please add a unit test to check if non-manager streams don't have metrics in fields to avoid such issues in the future?

@tolik0 test was added

**Review by tolik0 — APPROVED** [review]

**Review by artem1205 — APPROVED** [review]

File: airbyte-integrations/connectors/source-google-ads/metadata.yaml [hunk 1]
```diff
@@ -44,6 +44,12 @@ data:
           This release upgrades the Google Ads API to the latest version (v15), which causes changes in several schemas.
           Users should refresh the source schema and reset affected streams after upgrading to ensure uninterrupted syncs.
         upgradeDeadline: "2024-01-12"
+      4.0.0:
+        message:
+          This release updates Ad Group stream schema. Field metrics.cost_micros is not available for manager accounts.
```
Comment (alafanechere) [inline airbyte-integrations/connectors/source-google-ads/metadata.yaml:49]: You mention that this field is not available for manager accounts, what if users are not using a manager accounts, can't they still consume it?
Could we make the ad_group stream schema dynamic according to the account type?
My concern is that the scope of this breaking change might be too broad: we're removing a field for all users while it's unavailable only for a subset of users.

Comment (alafanechere) [reply airbyte-integrations/connectors/source-google-ads/metadata.yaml:49]: Cool! It'll remain a breaking changes for a subset of users but the scope is less broad 👍

**Review by artem1205 — APPROVED** [review]

**Review by tolik0 — APPROVED** [review]

**Pull request merged**